src/tests/Eip1993Factories.test.ts

```diff
@@ -0,0 +1,38 @@
+import {HTTPProvider} from '@zilliqa-js/core';
+import {Eip1993Factories} from '../utils/Eip1993Factories';
+import nock from 'nock';
+
+beforeEach(() => {
+  nock.enableNetConnect();
+  jest.restoreAllMocks();
+});
+
+describe('Eip1993Factories', () => {
+  describe('fromZilliqaProvider', () => {
+    it('should create provider from zilliqa provider', async () => {
+      const zilliqaProvider = new HTTPProvider('https://api.zilliqa.com');
+      const provider = Eip1993Factories.fromZilliqaProvider(zilliqaProvider);
+      expect(provider).toBeDefined();
+    });
+    it('should make request from created provider', async () => {
+      const zilliqaProvider = new HTTPProvider('https://api.zilliqa.com');
+      const provider = Eip1993Factories.fromZilliqaProvider(zilliqaProvider);
+      const resp = (await provider.request({
```

Comment:
All network methods should be conditionally mocked (should be mocked if tests wasn't run in LIVE mode). 

Reference: `src/tests/Resolution.test.ts:174`

Tests should work in live and non-live modes.